ideascube/tests/test_models.py

```diff
@@ -72,10 +72,19 @@ def test_user_data_fields_should_return_labels_and_values(settings):
 class JSONModel(models.Model):
     class Meta:
         app_label = 'ideascube'
+        managed = False
 
     data = JSONField()
 
 
+@pytest.fixture()
+def json_table():
+    from django.db import connections
+    cursor = connections['default'].cursor()
+    cursor.execute("CREATE TABLE ideascube_jsonmodel "
+                   "(id, data)")
```

Comment:
*Putting my comment here even though it is not related to this line, but there's no place comment on a commit message.*

> If we run tests with migration, django fail to correctly migrate the json table.

What? Why would it even try to migrate it?